### PR TITLE
Generate documentation with Skydoc.

### LINKED ISSUE #12: Document rules using Skydoc
Let's document the Rust rules using [Skydoc](https://github.com/bazelbuild/skydoc) and serve the HTML docs to make the easier to browse.

### PR DESCRIPTION
**Pull request by davidzchen**
Fixes #12

### CODE REVIEW

**Comment by davidzchen** [comment]
The CI failure is due to the fact that we are using tag 0.0.1 of rules_sass in skydoc, and we haven't cut a new release after the fix for adding cfg was checked in.

Let's merge bazelbuild/rules_sass#7, cut a release, update skydoc to use the new version of rules_sass, cut a new release of skydoc, and update this PR.

**Comment by damienmg** [comment]
I merged the rules_sass one, feel free to ping again when this PR is ready
for review.

**Comment by davidzchen** [comment]
Thanks, Damien. We still need a new release of skydoc with the recent fixes. The build is failing due to using an old version of protobuf. Sent bazelbuild/skydoc#41

**Comment by davidzchen** [comment]
Tests are passing. PTAL

File: WORKSPACE [hunk 1]
```diff
@@ -2,3 +2,20 @@ workspace(name = "io_bazel_rules_rust")
...
+)
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_repositories")
+sass_repositories()
+
+git_repository(
+    name = "io_bazel_skydoc",
+    remote = "https://github.com/bazelbuild/skydoc.git",
+    tag = "0.1.2",
+)
+load("@io_bazel_skydoc//skylark:skylark.bzl", "skydoc_repositories")
```
Comment (damienmg) [inline WORKSPACE]: nit: space before load (also upper).

maybe pass buildifier on this file?

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
+
```
Comment (damienmg) [inline docs/index.html]: Shouldn't that file be only on the site branch? Or maybe only have the readme checked in that redirect to the website?

File: WORKSPACE (hunk 1 above)
Comment (davidzchen) [inline WORKSPACE]: Done. Ran buildifier.

File: docs/index.html (hunk 2 above)
Comment (davidzchen) [inline docs/index.html]: You can publish from a directory in the master branch now. :)

https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/

Comment (damienmg) [inline docs/index.html]: :+1:

**Pull request merged**